metricflow/sql_clients/starrocks.py

```
@@ -0,0 +1,204 @@
+                for value in row:
+                    if pd.isna(value):
+                        values.append("NULL")
+                    elif isinstance(value, str):
+                        escaped_value = value.replace("'", "''")
+                        values.append(f"'{escaped_value}'")
+                    elif isinstance(value, (int, float)):
+                        values.append(str(value))
+                    elif isinstance(value, bool):
+                        values.append("1" if value else "0")
+                    else:
+                        values.append(f"'{str(value)}'")
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)